tools/toolchains/arm.py

```diff
@@ -404,8 +404,8 @@ def __init__(self, target, *args, **kwargs):
             "Cortex-M4F": "cortex-m4",
             "Cortex-M7F": "cortex-m7",
             "Cortex-M7FD": "cortex-m7",
-            "Cortex-M33": "cortex-m33+no_dsp+no_fp",
-            "Cortex-M33F": "cortex-m33+no_dsp",
+            "Cortex-M33": "cortex-m33+nodsp",
```

Comment:
no_fp was removed as well as the fix to no_dsp, is this intentional?

Reply:
Yes nofp as well gave error when used with clang. No_fp is valid with armlink

Reply:
Just wanted to make sure as this wasn't mentioned in the commit comment or PR description.